src/cli/run.js

```diff
@@ -26,6 +26,11 @@ export const builder = yargs => {
         alias: 's',
         description: 'display smaller ASCII QR code which could be unreadable in some fonts',
         type: 'boolean'
+      },
+      clean: {
```

Comment:
Think of these command-entry files as the documentation for the given command.
It's easier to see (and modify) the command arguments if every option is just written there.